src/style.css

```diff
@@ -0,0 +1,152 @@
+/* #region Normilized */
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
move border-box to * selector

move all reset and normalize styles to separate file